rpc/grpc/config.go

```diff
@@ -29,6 +29,10 @@ type Config struct {
 	// Endpoint is an address of GRPC netListener
 	Endpoint string `json:"endpoint"`
 
+	// Type of Socket defaults to "tcp" if unset
```

Comment:
You should also mention that `It must be "tcp", "tcp4", "tcp6", "unix" or "unixpacket".`, just like in [net.Listen]https://golang.org/pkg/net/#Listen).